src/general/OptionsMenu.tscn

```diff
@@ -938,28 +939,35 @@ text = "USE_A_CUSTOM_USERNAME"
 
 [node name="HBoxContainer3" type="HBoxContainer" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
 margin_top = 312.0
-margin_right = 511.0
+margin_right = 651.0
 margin_bottom = 339.0
 
 [node name="Label" type="Label" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer/HBoxContainer3"]
 margin_top = 2.0
-margin_right = 185.0
+margin_right = 184.0
 margin_bottom = 24.0
 text = "CUSTOM_USERNAME"
 
 [node name="HSeparator2" type="HSeparator" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer/HBoxContainer3"]
-margin_left = 189.0
-margin_right = 207.0
+margin_left = 188.0
+margin_right = 347.0
 margin_bottom = 27.0
 size_flags_horizontal = 3
 custom_styles/separator = SubResource( 5 )
 
 [node name="CustomUsername" type="LineEdit" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer/HBoxContainer3"]
-margin_left = 211.0
-margin_right = 511.0
+margin_left = 351.0
+margin_right = 651.0
 margin_bottom = 27.0
 rect_min_size = Vector2( 300, 0 )
 
+[node name="Screeshot Folder" type="Button" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
```

Comment:
The name shouldn't be spaced to make it consistent with other node naming. Also I'd prefer setting the horizontal size flag for the button to none, so it wouldn't become unnecessarily too wide.